scout/adapter/mongo/case.py

```diff
@@ -850,13 +849,38 @@ def update_case(self, case_obj, keep_date=False):
                     "sv_rank_model_version": case_obj.get("sv_rank_model_version"),
                     "track": case_obj.get("track", "rare"),
                     "updated_at": updated_at,
-                    "variants_stats": case_obj.get("variants_stats"),
                     "vcf_files": case_obj.get("vcf_files"),
                 },
             },
             return_document=pymongo.ReturnDocument.AFTER,
         )
 
+        # Remove non-mandatory key/values if they contain a null value
+        unset_keys = {}
+        for key in [
+            "custom_images",
+            "cnv_report",
+            "coverage_qc_report",
+            "gene_fusion_report",
+            "gene_fusion_report_research",
+            "mme_submission",
+            "multiqc",
+            "rank_model_version",
+            "smn_tsv",
+            "sv_rank_model_version",
+        ]:
+            if updated_case.get(key):  # Do not remove key if it has a value
+                continue
+            unset_keys[key] = 1
```

Comment:
I might have picked dict[key]="true", or just added the key to a set/list, but this works!

Reply:
I think you can also use "" or whatever you like apparently, as the value shouldn't affect the operation: https://docs.mongodb.com/manual/reference/operator/update/unset/

Reply:
I've changed to the value from 1 to "" now, so it's the same as in mongodb docs!

Reply:
> I've changed to the value from 1 to "" now, so it's the same as in mongodb docs!

The updated command works: 

<img width="1284" alt="image" src="https://user-images.githubusercontent.com/28093618/153816321-4087e9d1-f778-4772-a8f9-f8878ccb741e.png">